plul01-sp2

LucPlus opened the pull request:
https://eso.vse.cz/~plul01/sp2/

LucPlus committed:
plul01-sp2

https://eso.vse.cz/~plul01/sp2/

nvbach91 commented:
- inputy by mely byt ve formulari
- pokud se hrac rozhodne zacit hrat tak aplikace by si ho mel zapamatovat az se vrati zpatky, tedy neco jako login/logout
- pridejte moznost vyberu mnoziny pokemonu pro hru - pokemon gen1, gen2, gen3, ... nebo vsechny
- pridejte tlacitko "nevim" a pokracovat na dalsiho pokemona
- spocitejte podobnost slova a akceptujte odpoved pokud slovo je z 90% podobne a vic, chceme zachytit preklepy
- do leaderboardu pridejte seznam uhodnutych pokemonu (jejich artworky + title se jmenem)
- ten siluhet nefunguje dobre kdyz uzivatel klikne na Play again
- pridejte moznost zobrazeni dalsich siluhetu jako napoveda - obcas se stava ze neni uplne snadne uhodnout pokemona pouze z jednoho obrazku

LucPlus committed:
version 2

Kvůli změně formátu uložení 'leaderboardu' do localStorage aplikace zlobí se staršími uloženými daty. Prosím vymažte vaše localStoragy pro tuto stránku, aby vše fungovalo v pořádku.

nvbach91 commented:
- neni videt uhodnute pokemony
- zobrazit jen nejnovejsich 6 uhodnutych,
- zobrazit vsechny v modalu
- neopakovat pokemony ktere uzjsem uhodl

LucPlus committed:
Last version sp-2
